File: RenACarProjectApi/Models/Vehicle.cs
using System;
namespace RenACarProjectApi.Models;

public class Vehicle
{
	public int VehicleId { get; set; }
	public string Brand { get; set; }
	public string Model { get; set; }
	public string Type { get; set; }
	public string? ImageUrl { get; set; }
	public string TransmissionType { get; set; }
	public string FuelType { get; set; }
	public decimal DailyRentalPrice { get; set; }
	public int SeatCount { get; set; }
	public int DoorCount { get; set; }
	public bool HasAirConditioning { get; set; }


}

File: rent-a-car-project/src/app/models/vehicle-model/vehicle.ts
export class Vehicle {
    vehicleId: number;
    brand: string;
    model: string;
    type: string;
    imageUrl?: string;
    transmissionType: string;
    fuelType: string;
    dailyRentalPrice: number;
    seatCount: number;
    doorCount: number;
    hasAirConditioning: boolean;
  
    constructor(
      vehicleId: number,
      brand: string,
      model: string,
      type: string,
      transmissionType: string,
      fuelType: string,
      dailyRentalPrice: number,
      seatCount: number,
      doorCount: number,
      hasAirConditioning: boolean,
      imageUrl?: string
    ) {
      this.vehicleId = vehicleId;
      this.brand = brand;
      this.model = model;
      this.type = type;
      this.transmissionType = transmissionType;
      this.fuelType = fuelType;
      this.dailyRentalPrice = dailyRentalPrice;
      this.seatCount = seatCount;
      this.doorCount = doorCount;
      this.hasAirConditioning = hasAirConditioning;
      this.imageUrl = imageUrl;
    }
  }
  